pkg/agent/openflow/pipeline.go

```diff
@@ -795,14 +795,20 @@ func (c *client) l3ToPodFlow(podInterfaceIPs []net.IP, podInterfaceMAC net.Hardw
 
 // l3ToGWFlow generates the flow to rewrite MAC to gw port if the packet received is unmatched by local Pod flows.
 // This flow is used in policy only traffic mode.
-func (c *client) l3ToGWFlow(gwMAC net.HardwareAddr, category cookie.Category) binding.Flow {
+func (c *client) l3ToGWFlow(gwMAC net.HardwareAddr, category cookie.Category) []binding.Flow {
 	l3FwdTable := c.pipeline[l3ForwardingTable]
-	return l3FwdTable.BuildFlow(priorityLow).MatchProtocol(binding.ProtocolIP).
-		Action().SetDstMAC(gwMAC).
-		Action().DecTTL().
-		Action().GotoTable(l3FwdTable.GetNext()).
-		Cookie(c.cookieAllocator.Request(category).Raw()).
-		Done()
+	ipProtos := c.getNodeIPProtocols()
```

Comment:
I saw in some other cases you use enableIPv4 and enableIPv6? Could we unify to the same way?

Could we make enableIPv4 and enableIPv6 two properties of client, and need not to pass them to all funcs?